themes/book/app/routes/$.tsx

```diff
@@ -102,7 +103,7 @@ function ArticlePageAndNavigationInternal({
       <PrimaryNavigation
         sidebarRef={toc}
         hide_toc={hide_toc}
-        footer={<MadeWithMyst />}
+        footer={<SidebarFooter content={projectParts?.primary_sidebar_footer?.mdast} />}
```

Comment:
As a follow-up, we may want to rename `footer` here to something more descriptive, since this does not refer to the main page footer.

Maybe `sidebarFooter`.

Reply:
IMO, ideally this would be primarySidebarFooter and it could live alongside secondarySidebarFooter one day. Or, call it something more generic like `primarySidebarEnd` to recognize that other things than footers might be placed there one day.

I think my suggestion would be we just try to re-use the JB1 structure naming as much as possible for the book theme, since it has been used and battle-tested for a long time now:

https://pydata-sphinx-theme.readthedocs.io/en/stable/user_guide/layout.html